test/predicates.jl

```diff
@@ -130,6 +130,14 @@
   end
 
   @testset "intersects" begin
+    function assertintersecsundertransforms(g1, gt, gf)
+      translations = Iterators.map(Translate, [(0, 0), (10, 0), (0, 10), (-10, 0), (0, -10)])
+      rotations = Iterators.map(Rotate ∘ Rotations.Angle2d, 0:(π / 4):(7π / 4))
+      transforms = (t ∘ r for (t, r) in Iterators.product(translations, rotations))
+      @test all(intersects(t(g1), t(gt)) for t in transforms)
+      @test all(!intersects(t(g1), t(gf)) for t in transforms)
```

Comment:
Let's rewrite this in terms of a simple for loop:

```julia
for t in Translate.([(), (), ...]), r in Rotate.([...})
  f = t \circ r
  @test intersects(f(g1), f(g2))
end

Reply:
In general, we try to copy/paste the tests to facilitate debugging later. When a test fails in a loop inside a function, it is hard to know which iteration failed precisely, so even loops we try to avoid, and instead copy/paste the iterations by hand.

In this case, it suffices to copy/paste the loop in the test directly. Ideally we would just pick example candidates from each octant, orthant.